dandi/utils.py

```diff
@@ -890,3 +891,66 @@ def joinurl(base: str, path: str) -> str:
         return path
     else:
         return base.rstrip("/") + "/" + path.lstrip("/")
+
+
+def get_retry_after(response: requests.Response) -> Optional[int]:
+    """If provided and parsed ok, returns duration in seconds to sleep before retry.
+
+    If not provided in the response header `Retry-After`, would
+    return None.
+    If parsing fails, or provided date/sleep does not make sense
+    since either too far in the past (over 2 seconds) or in the future
+    (over a week), would return None.
+    """
+    if_unparsable = None
```

Comment:
This is basically a constant alias.  What's the point of this assignment?  Just get rid of it and change `sleep_amount = if_unparsable` to `sleep_amount = None` (or even `return None`).

Reply:
FTR: initially I had it as an option, then saw that I never used it, so moved "into body". Ok, will change as recommended to simplify